marimbabot_vision/launch/marimbabot.launch

```diff
@@ -3,5 +3,13 @@
      <rosparam>
        cv_camera/device_id: 0
      </rosparam>
+      <rosparam>
+       cv_camera/rate: 1
+     </rosparam>
+</node>
+<node name="vision_node" pkg="marimbabot_vision" type="vision_node.py">
+     <rosparam>
```

Comment:
I am not familiar with this notation for parameters. I would suggest a config folder in the package. We can place a yaml file with all parameters there and load it in the launch file

Reply:
I don't know what you mean by that exactly. If you want you can create a pull request for that. :)

Reply:
Like this: https://github.com/bit-bots/bitbots_navigation/blob/848da868363ccd4201f6cbd0c86d1ce28bb8f69f/bitbots_localization/launch/localization.launch#L13
Which loads this parameter file:
https://github.com/bit-bots/bitbots_navigation/blob/ros1/bitbots_localization/config/config.yaml

Reply:
In the case I don't do it in near future. Would you mind merging this first and adjusting that later? I could create a new PR for that.